components/about/component_workflow.md

```diff
@@ -0,0 +1,36 @@
+---
+description: Components are constantly being added to the design system. The backlog shows what’s on the list, upcoming, and currently in development.
+title: Component Backlog
+status: draft
+---
```

Comment:
are these documents meant to be apart of devhub?

Reply:
I believe these will eventually go under the about section of the design system. @bl-smith can't remember what you decided last week about making these public on DevHub or just keeping them on GitHub.

Reply:
Thanks for looking at this, @patricksimonian  - the devhubignore file does have the line "docs/*.md" - doesn't that meant that if we put the Change Request Process file in that folder it will be ignored? The other two files should get pulled into Dev Hub, so no worries there.

Reply:
yes it will :)  but this file appears to be in /components/... right?

Reply:
I may be having the case of the mondays

Reply:
Ok makes sense.

I can add the ignore:true to the front matter of the change request process.

Reply:
ohh i misread my bad. Anyhow..bryan if you could add `author: bl-smith` to the front matter of the docs you've written. We have added support for avatars to the devhub. It'll help tart the whole thing up.

Reply:
@orinevares  and @ksingbeil - maybe we could talk further about how we want our avatars to appear and where. I notice that the GovUK uses their generic Github account (govuk-design-system) to publish documentation on Github. This makes sense to me, because if someone has a question about the design system processes or administration, they should contact the Design System Administrators, not necessarily me. Should we do something similar?

Reply:
@orinevares @bl-smith @ksingbeil good points all, if you would prefer the avatar to be just bcgov. The `author: orinevares` attribute (for example) can just be switched to `author: bcgov`

Reply:
The images is open to the public via GitHub. If people don't want it to be public than for consistency it should be removed from GitHub and replaced with a non-personal avatar. Otherwise it should be left in.

Reply:
@patricksimonian BCGov feels too generic. I think a Design System avatar would be good for documentation...but I'm not sure the logistics on how this connects with Git - does this mean there needs to be a generic Design System Git account? I think this treads into what @jleach is talking about.

Reply:
Hmm, this is actually leading to a support issue on how to 'display an avatar' in devhub. The technical aspects right now is that we are assuming the 'author' is a github user, that way we can easily grab the avatar.

So I'll have to answer the questions in two ways.

First yes, techinically *at this time* a design system github account would be needed, the 'author' attributes for all markdown files would have to be changed

Second possible answer, this is actually begging the question for the ability to add a avatar image in the configuration. No promises on that as it requires a bit of research into some web standards on what can be [Unfurled](https://medium.com/slack-developer-blog/everything-you-ever-wanted-to-know-about-unfurling-but-were-afraid-to-ask-or-how-to-make-your-e64b4bb9254) from content.

Reply:
Hi All,

Seems like this thread has taken a new direction and raised a further conversation. For the sake of closing this PR I will mark this as resolved but with the understanding to investigate the avatar feature further.